.github/workflows/main.yml

```
@@ -0,0 +1,48 @@
+name: RHVoice-CI
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  androidBuild:
+    name: Build android app
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Assemble app debug APK
+        working-directory: ./src/android
+        run: sudo ./gradlew assembleDebug --stacktrace
+
+      - name: Upload app APK
+        uses: actions/upload-artifact@v1
+        with:
+          name: app-debug
+          path: src/android/RHVoice-core/build/outputs/apk/debug/RHVoice-core-debug.apk
+
+  windowsBuild:
+    name: Build windows packages
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip lxml scons
+      - name: build
+        run: scons
```

```suggestion
        run: scons -j 2
```

let's use 2 CPUs. that's faster.

---

Done